xbmc/cores/RetroPlayer/RetroPlayer.cpp

```diff
@@ -489,6 +493,37 @@ std::string CRetroPlayer::GameClientID() const
   return "";
 }
 
+std::string CRetroPlayer::GetPlayingGame() const
+{
+  if (m_gameClient)
+    return m_gameClient->GetGamePath();
+
+  return "";
+}
+
+IPlayback* CRetroPlayer::GetPlayback() const
+{
+  return m_playback.get();
+}
+
+void CRetroPlayer::ResetGame()
+{
+  if (m_gameClient)
+  {
+    float speed = static_cast<float>(m_playback->GetSpeed());
```

Comment:
Unused variable. You could set the speed back after hardware reset, or does hardware reset force speed to zero? maybe it shouldn't do that?